.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: c
+sudo: required
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+dist: bionic
+cache:
+    directories:
+    - $HOME/.opam
+env:
+  global:
+    - DEPOPTS="*"
+    - TESTS=true
+    - DISTRO=ubuntu-18.04
+    - POST_INSTALL_HOOK="opam reinstall --with-test kafka && make integration"
+before_install:
+    - wget http://mirrors.dotsrc.org/apache/kafka/2.4.0/kafka_2.12-2.4.0.tgz -O kafka.tar.gz
+    - mkdir -p kafka && tar xzf kafka.tar.gz -C kafka --strip-components 1
+    - kafka/bin/zookeeper-server-start.sh kafka/config/zookeeper.properties &
+    - sleep 3
+    - kafka/bin/kafka-server-start.sh kafka/config/server.properties &
+    - sleep 5
+    - kafka/bin/kafka-topics.sh --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 2 --topic test
+matrix:
+  include:
+  - os: linux
+    env: OCAML_VERSION=4.02 PACKAGE="kafka"     PINS="kafka:."
+  - os: linux
+    env: OCAML_VERSION=4.02 PACKAGE="kafka_lwt"
+  - os: linux
+    env: OCAML_VERSION=4.09 PACKAGE="kafka"     PINS="kafka:."
+  - os: linux
+    env: OCAML_VERSION=4.00 PACKAGE="kafka_lwt" PINS="kafka:. kafka_lwt:."
```

Comment:
This should be 4.09. The corresponding build fails to compile `dune`. These are likely related.